src/main/java/com/puppycrawl/tools/checkstyle/utils/CodePointUtil.java

```diff
@@ -0,0 +1,47 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2021 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.utils;
+
+/**
+ * Contains String Utility methods.
+ *
+ */
+public final class CodePointUtil {
+
+    /** Stop instances being created. **/
+    private CodePointUtil() {
+    }
+
+    /**
+     * Returns an array that is sub array of this array. The sub array begins
+     * at beginIndex and ends at endIndex
+     *
+     * @param codePoints Array
+     * @param beginIndex - the beginning index, inclusive
+     * @param endIndex - the ending index, exclusive
+     * @return the specified sub array
+     */
+    public static int[] subArray(int[] codePoints, int beginIndex, int endIndex) {
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/util/Arrays.html#copyOfRange(int[],%20int,%20int)

?

Reply:
Yes, didn't realize. Will do the change

Reply:
Done